pipelines/DRP.yaml

```diff
@@ -1,16 +1,158 @@
-description: DRP specialized for DECam
+description: The DRP pipeline specialized for the DECam instrument
 instrument: lsst.obs.decam.DarkEnergyCamera
 imports:
-  - location: $PIPE_TASKS_DIR/pipelines/DRP.yaml
-    exclude:
-      - sourceTable
-      - objectTable
-      - isr
-  - location: $AP_PIPE/pipelines/RunIsrWithCrosstalk.yaml
-# This DRP pipeline uses an obs_decam specific ISR configuration that
-# requires crosstalk sources to be pre-generated via the
-# RunIsrForCrosstalkSources.yaml pipeline.  This applies overscan
-# correction and writes a new dataset that is then used as needed for
-# the inter-chip crosstalk solutions.  Unless a specific overscan
-# change is required, it may be best to generate these crosstalk
-# sources soon after ingest to allow them to be readily available.
+  location: $PIPE_TASKS_DIR/pipelines/DRP.yaml
+tasks:
+  isrForCrosstalkSources:
```

Comment:
Unless something has changed, this cannot be run in the same pipeline that regular ISR is run in. Something about the output dataset types being identical and the quantum graph having no way to disambiguate. I'm afraid you'll need a two-step pipeline like `ap_pipe` has.

Reply:
Hm, perhaps something has changed? I have been using this to reduce Merian DECam data without issue? To be clear, I've not simply been running the entire DRP.yaml file, as that wouldn't make sense (as it doesn't make sense in `obs_subaru` or `imsim` either, because different tasks require different data ID inputs). Rather, I've been running `step0`, `step1`, `step2`, `step3` and `step7` subsets, all without issue.

Reply:
For the record, the crosstalk integration issue is described on [DM-31492](https://jira.lsstcorp.org/browse/DM-31492). However, if the DRP practice is to treat each subset as its own pipeline, then it shouldn't matter.

Reply:
Having spoken with @czwa about this today, I can confirm that running this pipeline in its entirety will result in a PrerequisiteInput error. However, as in `obs_subaru`, this YAML file is not designed to be run from the top level. Rather, it is intended to be run using the subsets or tasks alone. The necessity to specify different data ID inputs to different steps of processing (single frame, multi band, etc) makes this a requirement.